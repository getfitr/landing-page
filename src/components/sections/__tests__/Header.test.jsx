import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Header from '../Header';

test('Shows the Header', () => {
  const header = renderer.create(
    <BrowserRouter>
      <Header />
    </BrowserRouter>
  );
  const tree = header.toJSON();
  expect(tree).toMatchSnapshot();
});