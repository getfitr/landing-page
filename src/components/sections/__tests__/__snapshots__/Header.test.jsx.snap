// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shows the Header 1`] = `
<nav
  className="css-10xmei8"
>
  <div
    className="css-70qvj9"
  >
    <div>
      LOGO
    </div>
  </div>
  <div
    className="css-dsyim5"
    onClick={[Function]}
  >
    <svg
      fill="none"
      height="1em"
      stroke="currentColor"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeWidth="2"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 24 24"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <line
        x1="3"
        x2="21"
        y1="12"
        y2="12"
      />
      <line
        x1="3"
        x2="21"
        y1="6"
        y2="6"
      />
      <line
        x1="3"
        x2="21"
        y1="18"
        y2="18"
      />
    </svg>
  </div>
  <div
    className="css-8olkqp"
  >
    <div
      className="css-prcoui"
    >
      <p
        className="chakra-text css-n7rqg5"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          GetFitr! App
        </a>
      </p>
      <p
        className="chakra-text css-n7rqg5"
      >
        <a
          href="/project"
          onClick={[Function]}
        >
          The GetFitr! Dev Project
        </a>
      </p>
      <p
        className="chakra-text css-1g5pnks"
      >
        <a
          href="/signup"
          onClick={[Function]}
        >
          <button
            className="chakra-button css-1uo5x05"
            type="button"
          >
            Create an Account
          </button>
        </a>
      </p>
    </div>
  </div>
</nav>
`;
